src/CMakeLists.txt

```diff
@@ -29,7 +29,13 @@ set(MORPHIO_SOURCES
   mut/writers.cpp
   mut/modifiers.cpp)
 
-file(STRINGS ${CMAKE_CURRENT_SOURCE_DIR}/../VERSION MORPHIO_VERSION_STRING)
+execute_process(COMMAND python setup.py --version
```

Comment:
debugging this will be interesting when it fails, if I understand this right:
cmake calls into setup.py, but it may already be running from setup.py in a bunch of sub-processes because it's called from setup.py

have you tested that this works from clean repository pull and such? I'm concerned that since the version is calculated from git, when working on this in the 'editable' manor, or as pip/setuptools changes, (like when it copies all the files required for building to a different folder) that we'll run into problems

Also, how does the versioning interact w/ spack?

Reply:
> Also, how does the versioning interact w/ spack?

Either we use the same Git tag for fetching source from the repo; or fetch source tarball from PyPI as in https://github.com/BlueBrain/spack/pull/372.

Reply:
Yes, I just tested, it seems to work when installing in editable mode or from the tarball.